test/cmd/apply.sh

```diff
@@ -79,7 +79,7 @@ run_kubectl_apply_tests() {
   ! grep -q emptyDir <<< "$(kubectl get deployments test-deployment-retainkeys -o yaml "${kube_flags[@]:?}")" || exit 1
   # Clean up
   kubectl delete deployments test-deployment-retainkeys "${kube_flags[@]:?}"
-
+  kubectl wait pod --for=delete --all --timeout=15s || true
```

Comment:
would it be simpler to change the get_object_assert that there are no pods below to wait_object_assert?

Reply:
done